src/extension.ts

```diff
@@ -293,6 +344,9 @@ function startWebhint(context: vscode.ExtensionContext): void {
             client.outputChannel.clear();
             client.outputChannel.show(true);
         });
+
+        // Once client is ready reload any present configuration.
+        client.sendNotification(reloadAllProjectsConfigNotification);
```

Comment:
Since this effectively restarts the language server anyway, could we do the following to avoid having an extra contract between the host and language server?
```ts
await stopWebhint();
startWebhint(context);
```